src/main/java/org/zalando/putittorest/annotation/RestClient.java

```diff
@@ -0,0 +1,26 @@
+package org.zalando.putittorest.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+@Target(ElementType.FIELD)
```

Comment:
But we can make it also useable for parameters. Then you can use it also this way:
```
@Component
public class Wrapper {

    private final Rest exampleRest;

    @Autowired
    public Wrapper(@RestClient("example") Rest exampleRest) {
        this.exampleRest = exampleRest;
    }

    public Rest rest() {
        return exampleRest;
    }
}
```
It works then like an Qualifier for that parameter.